doc/2/core-classes/search-result/next/snippets/fromsize-koncorde.dart

```diff
@@ -0,0 +1,23 @@
+final List<Map<String, dynamic>> documents = [];
+
+for (var i = 0; i < 100; i++) {
+  documents.add({ '_id': 'suv_no${i}', 'body': { 'category': 'suv' } });
+}
+
+await kuzzle.document.mCreate('nyc-open-data', 'yellow-taxi', documents,
+  waitForRefresh: true
+);
+
+var res = await kuzzle.document.search(
+  'nyc-open-data',
+  'yellow-taxi',
+  query: { 'query': { 'equals': { 'category': 'suv' } } },
```

Comment:
Same here, why we have two times `query` ?

Reply:
As we discussed, this is normal here